jhhalchemy/tests/integration/conftest.py

```diff
@@ -0,0 +1,62 @@
+"""
+pytest fixtures for setting up a DB connection and creating tables to verify the models' functionality.
+
+To use this, you need to set MYSQL_CONNECTION_URI.
+The account you connect with should be able to:
+1. create database
+2. create a table in that database and have all access rights to it
+3. drop the created database
+
+Note: this might work with other databases besides mysql/maria, but we haven't tested that.
+"""
+import flask
+import flask_sqlalchemy
+import pytest
+import sqlalchemy
+import sqlalchemy_utils
+
+#
+# Set your own MYSQL connection string here
+#
+MYSQL_CONNECTION_URI = 'mysql://root:root@0.0.0.0/test_db'
```

Comment:
Parameterize credentials for testings.

Reply:
Good idea. Allowing override with an ENV var.